Notebooks/beavis-ci.csh

```diff
@@ -0,0 +1,168 @@
+#! /usr/bin/env tcsh
+#=======================================================================
+#+
+# NAME:
+#   beavis-ci.csh
+#
+# PURPOSE:
+#   Enable occasional integration and testing. Like travis-ci but dumber.
+#
+# COMMENTS:
+#   Makes "rendered" notebooks and deploys them to a "rendered" orphan 
+#   branch, pushed to GitHub for web display. 
+#
+# INPUTS:
+#
+# OPTIONAL INPUTS:
+#   -h --help     Print this header
+#   -u --username GITHUB_USERNAME, defaults to the environment variable
+#   -k --key      GITHUB_API_KEY, defaults to the environment variable
+#   -n --no-push  Only run the notebooks, don't deploy the outputs
+#   --html        Make html outputs instead
+#
+# OUTPUTS:
+#
+# EXAMPLES:
+#
+#   ./beavis-ci.csh -u drphilmarshall -k a0573fc078b7e8f5c10660e1f29e58678bf4d118
```

Comment:
GitHub detected this line, and promptly disabled my API token. Ah well, the user will get the picture, hopefully.